libraries/AC_Fence/AC_PolyFence_loader.cpp

```diff
@@ -1158,8 +1158,8 @@ bool AC_PolyFence_loader::get_return_point(Vector2l &ret)
         }
     }
 
-    ret.x = ((min_loc.x+max_loc.x)/2);
-    ret.y = ((min_loc.y+max_loc.y)/2);
+    ret.x = (min_loc.x/2) + (max_loc.x/2);
```

Comment:
Please add a comment. Someone reading these lines latter may ask why the strange math here. :)